tests/lib/config.js

```diff
@@ -358,9 +358,6 @@ describe("Config", () => {
                         browser: true,
                         node: false
                     },
-                    ecmaFeatures: {
-                        globalReturn: false
```

Comment:
globalReturn is still a valid ecmaFeature, isn't it? (If we just don't need it, though, that's a different story.)